### DISCUSSION TITLE
Table segmentation new stretch rule

### DISCUSSION THREAD

**Discussion by jayeshthukarul** [deepdoctection/deepdoctection#109] [General]
Can we add new stretch logic into the table segmentation service. Current rules are "left" and "equal ". "left" : stretches column to the right of left neighboring column. "equal" : brings it to center. In similar fashion we can add "right" : which will make column right edge shift to right edge left column. Is this possible?

**Comment by JaMe76** [comment]
Sure, there is nothing special about stretching "left" as opposed to "right". Let me see, I think I could add this feature the next release

**Reply by jayeshthukarul** [reply]
Yes. In one of my project particularly related to company annual reports "right" rule is working best for columns. I have coded it and tested it.
